yarn.lock

```diff
@@ -6148,6 +6148,14 @@ chalk@^4.0.0, chalk@^4.1.0:
     ansi-styles "^4.1.0"
     supports-color "^7.1.0"
 
+chalk@^4.1.1:
+  version "4.1.2"
+  resolved "https://registry.yarnpkg.com/chalk/-/chalk-4.1.2.tgz#aac4e2b7734a740867aeb16bf02aad556a1e7a01"
+  integrity sha512-oKnbhFyRIXpUuez8iBMmyEa4nbj4IOQyuhc/wy9kY7/WVPcwIO9VA668Pu8RkO7+0G76SLROeyw9CpQ061i4mA==
+  dependencies:
+    ansi-styles "^4.1.0"
+    supports-color "^7.1.0"
+
```

Comment:
I wish yarn was more aggressive at bumping minor of existing deps.

Reply:
Lots of dupes, yeah. I added a `yarn-deduplicate` commit 9bf8751cbedd52c12326480d0bf700e66a4dab3a

Reply:
Sweet!